cmd/templ/lspcmd/proxy/server.go

```diff
@@ -346,18 +347,19 @@ func (p *Server) Completion(ctx context.Context, params *lsp.CompletionParams) (
 			if !ok {
 				continue
 			}
-
 			te := handleImportTextEdit(doc, item.Detail)
 			item.AdditionalTextEdits = []lsp.TextEdit{te}
-		}
-		if strings.Contains(item.Detail, "(from \"") {
-			doc, ok := p.TemplSource.Get(string(templURI))
-			if !ok {
-				continue
+		} else {
+			m := completionWithImport.FindStringSubmatch(item.Detail)
```

Comment:
it seems that on the go side, any kind besides module can have the package embedded in the detail field 

[also it seems that only completion](https://github.com/golang/tools/blob/21090a2aa8d3719d5a5d4264e41696529522f0bd/gopls/internal/lsp/source/completion/format.go#L179)

I think this should do it. Let me know what do you think